src/SwiftUI/SwiftUI/ModifiedOpacity.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Swift;
+using Swift.Interop;
+
+namespace SwiftUI
+{
+    public class ModifiedOpacity<T> : View where T : View
+    {
+        public static IntPtr Size;
+
+        public unsafe static SwiftType SwiftType
+        {
+            get
+            {
+                var arrayArgumentsPointer = stackalloc void*[1];
+                arrayArgumentsPointer[0] = SwiftType.Of(typeof(T))!.Metadata;
+                var opaqueTypeMetadata = SwiftCoreLib.GetOpaqueTypeMetadata(0, arrayArgumentsPointer, SwiftUILib.Types.ViewOpacityTypeDescriptor, 0);
+                var witnessTable = (ValueWitnessTable**)(((void**)opaqueTypeMetadata) - 1);
+                Size = (**witnessTable).Size;
+                return new SwiftType(new IntPtr(opaqueTypeMetadata));
+            }
+        }
+
+        public ModifiedOpacity(TaggedPointer taggedPointer, SwiftType viewType) : base(taggedPointer, viewType)
```

Comment:
This constructor should be internal
```suggestion
        internal ModifiedOpacity (TaggedPointer taggedPointer, SwiftType viewType) : base (taggedPointer, viewType)
```

Reply:
Agreed. Done!